tuf-spec.md

```diff
@@ -822,23 +825,27 @@ repo](https://github.com/theupdateframework/specification/issues).
    TARGETPATH.  The application may use this information to guide download
    decisions.
 
-   DELEGATIONS is an object whose format is the following:
+   "keys_for_delegations" lists the public keys to verify signatures of
+   delegated targets roles. Revocation and replacement of delegated targets
+   roles keys is done by changing the keys in this field in the delegating
+   role's metadata.
 
-       { "keys" : {
-             KEYID : KEY,
-             ... },
-         "roles" : [{
-             "name": ROLENAME,
-             "keyids" : [ KEYID, ... ] ,
-             "threshold" : THRESHOLD,
-             ("path_hash_prefixes" : [ HEX_DIGEST, ... ] |
-              "paths" : [ PATHPATTERN, ... ]),
-             "terminating": TERMINATING,
-         }, ... ]
-       }
+   "delegations" is a list of DELEGATION objects whose format is the following:
 
-   ROLENAME is the name of the delegated role.  For example,
-   "projects".
+        {
+          "name": DELEGATION_NAME,
+          ("path_hash_prefixes" : [ HEX_DIGEST, ... ] |
+          "paths" : [ PATHPATTERN, ... ]),
+          "terminating": TERMINATING,
+          "min_roles_in_agreement" : NUM_ROLES,
```

Comment:
This was probably named `min_roles_in_agreement` to distinguish it from the signature `threshold` field. But I find it  odd that one threshold field  has such a specific name and the other one is generic. 

Maybe the scope (delegation vs. role) is disambiguation enough and we can both call `threshold`?

Reply:
maybe delegation_threshold?

Reply:
The field is a property of the `delegation` object. Addressing it as `delegation.delegation_threshold` seems like unnecessary redundancy.

Reply:
I agree. Let's just call it `threshold`.

Reply:
That was the reasoning, yes (distinguishing it from the other `threshold`).  If you think it'll not be confused, then it's nice to set it to something simple.

Reply:
Thinking about it some more, it does make sense to distinguish it from the other threshold. Ideally, we'd have equally specific names for both, e.g. `role.min_keys_in_agreement` and `delegation.min_roles_in_agreement` or `role.signature_threshold` and `delegation.role_threshold`.

 Is renaming the existing threshold field an option? Is it worthwhile? This will be a breaking change anyway. :)

Until then I'll just leave it as `threshold` and `min_roles_in_agreement`.